### PR TITLE
Kubevirt addon

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: scripts/Manifest [hunk 1]
```diff
@@ -4,4 +4,17 @@ INSTALLER_ID=
 REPLICATED_APP_URL=https://replicated.app
 KURL_UTIL_IMAGE=replicated/kurl-util:alpha
 KURL_BIN_UTILS_FILE=
-INSTALLER_YAML=
+INSTALLER_YAML="apiVersion: cluster.kurl.sh/v1beta1
```
Comment (mzaneri) [inline scripts/Manifest:7]: this should not be changed

File: kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go [hunk 2]
```diff
@@ -40,6 +40,8 @@ type InstallerSpec struct {
 	FirewalldConfig FirewalldConfig `json:"firewalldConfig,omitempty"`
 	Ekco            Ekco            `json:"ekco,omitempty"`
 	Containerd      Containerd      `json:"containerd,omitempty"`
+	Kubevirt        Kubevirt        `json:"kubevirt,omityempty"`
```
Comment (mzaneri) [inline kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go:43]: have a dupe in installer types here, line 43 should be deleted

File: scripts/Manifest [hunk 3]
```diff
@@ -4,4 +4,4 @@ INSTALLER_ID=
 REPLICATED_APP_URL=https://replicated.app
 KURL_UTIL_IMAGE=replicated/kurl-util:alpha
 KURL_BIN_UTILS_FILE=
-INSTALLER_YAML=
+INSTALLER_YAML=
```
Comment (mzaneri) [inline scripts/Manifest:7]: not sure if no newline with cause an issue

**Review by mzaneri — DISMISSED** [review]
lgtm

**Comment by DanStough** [comment]
Closing as stale.

**Pull request closed without merging**